src/index.js

```diff
@@ -50,6 +54,8 @@ export function parseValue(value) {
     return parseList(value);
   } else if (_.isPlainObject(value)) {
     return parseMap(value);
+  } else if (ESCAPE_STRINGS && _.isString(value)) {
```

Comment:
Is there a better way than to put e.g. `var options;` in the outside scope and use that in `parseValue`? 😄